doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -130,40 +119,41 @@ The following are the prefixes used to indicate the cores:
 Assisted multicore logging
 ==========================
 
-.. to review
+Assisted multicore logging uses dictionary-based logging to send messages without redundant strings to STM, and is based on the :ref:`zephyr:logging_guide_dictionary` feature of the logging API provided by Zephyr.
+For more information on assisted multicore logging, see :ref:`zephyr:logging_cs_stm`.
 
-Assisted multicore logging uses dictionary-based logging to send messages without redundant strings to STM, and is based on the :ref:`logging_guide_dictionary` feature of the logging API provided by Zephyr.
-Instead of including the format strings in the log messages, it logs the addresses (message IDs) where the strings are stored, which reduces the size of the logging subsystem.
-If the data goes to the ETR buffer, the proxy core's responsibility is to dump this data.
-The host PC, equipped with a decoder tool, translates these addresses back into human-readable text using a JSON database generated during the build process.
+Configuring logging
+-------------------
 
-When using logs, this method has the following advantages:
+To show the logs for a given domain in the UART output, you must enable the STM for both the application core (as the ETR buffer processing proxy) and that domain's CPU.
+To do so, use the :ref:`zephyr:nordic-log-stm-dict` snippet when building the application for the related cores.
+For example, the following command uses the snippet when building for the application core::
 
-* It reduces the size of the binary, as the strings used in the log messages are not stored in the binary itself.
-* It reduces the amount of data that needs to be sent to and processed by the application core, as the string formatting is offloaded to the host side.
+   west build -b nrf54h20dk/nrf54h20/cpuapp -S nordic-log-stm-dict
 
-Configuration
--------------
+All cores must use the same logging configuration.
 
-.. to review
-
-To enable the assisted multicore logging for a domain, set the :kconfig:option:`CONFIG_LOG_STM_DICT` Kconfig option for that domain's image to ``y``.
-
-After building your application, a dictionary database file named :file:`log_database.json` will be generated in the :file:`build/zephyr` directory.
+After building your application, a dictionary database file named :file:`log_database.json` will be generated in the build directories for each one of the cores the snippet was used on (:file:`build/<app_name>/zephyr/`, where ``<app_name>`` is the name of the application built for each specific core).
 This file is crucial for decoding the logs and is updated with every build.
 
 Reading the logs
 ----------------
 
-.. to review
-
 To read the dictionary-based STM log output, do the following:
 
 1. Set up the log capture.
 
    Use the ``nrfutil trace stm`` command to start capturing logs from the device, specifying the database configuration for each domain ID, as well as the serial port, the baud rate, and the output file name::
```

Comment:
You can also link to https://docs.nordicsemi.com/bundle/nrfutil/page/nrfutil-trace/nrfutil-trace_2.10.0.html#nrfutil-trace-stm-usage-examples that will be available soon.
More docs will be available at a later date: https://nordicsemi.atlassian.net/browse/NRFU-1247